src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final int MIN_PASSWORD_LENGTH = 10;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null && repeatPassword != null
+                && (password.equals(repeatPassword)
+                && password.length() >= MIN_PASSWORD_LENGTH)) {
+            return true;
+        } else {
```

Comment:
again: do we need `else` after `return`? please answer

Reply:
I think yes, because if conditions in the block "if" will return "false", this "return" will not work, and then I throw new exception